Ports/AvailablePorts.md

```diff
@@ -85,6 +85,7 @@ Please make sure to keep this list up to date when adding and updating ports. :^
 | [`libphysfs`](libphysfs/)              | PhysicsFS                                                       | 3.0.2                    | https://icculus.org/physfs/                                                    |
 | [`libpng`](libpng/)                    | libpng                                                          | 1.6.37                   | https://libpng.org/                                                            |
 | [`libpuffy`](libpuffy/)                | libpuffy                                                        | 1.0                      | https://github.com/ibara/libpuffy                                              |
+| [`libssh2`](libssh2/)                  | libssh2                                                         | 1.10.0                   | https://www.libssh2.org/                                                       |
```

Comment:
Just a small nitpick.
This list should be sorted alphabetically so this should be just a little lower

Reply:
Fixed it! Thanks :)